easy-ui-react/src/Notification/Notification.tsx

```diff
@@ -0,0 +1,374 @@
+* import { Button } from "@easypost/easy-ui/Button";
+*
+* function Component() {
+*  const notification = useNotification();
+*
+*  const onToast = () => {
+*    notification.showPromotionalToast({ message: "message", hasIcon: false });
+*  };
+*
+*  const onAlert = () => {
+*    notification.showWarningAlert({ message: "message", hasIcon: false });
+* };
+*
+*  return (
+*    <>
+*      <Button onPress={onToast}>Show toast</Button>
+*      <Button onPress={onAlert}>Show alert</Button>
+*    </>
+*  );
+*}
+*```
+* @example
+* _With onDismiss callback, only applicable for alerts:_
+```tsx
+* import { useNotification } from "@easypost/easy-ui/Notification";
+* import { Button } from "@easypost/easy-ui/Button";
+*
+* function Component() {
+*  const notification = useNotification();
+*
+*
+*  const onAlert = () => {
+*    notification.showNeutralAlert({
+*      message: "message",
+*      onDismiss: () => alert("i am called immediately before the alert closes"),
+*    });
+*  };
+*
+*  return (
+*    <>
+*      <Button onPress={onAlert}>Show alert</Button>
+*    </>
+*  );
+*}
+*```
+ */
+export function useNotification() {
+  const notification = useContext(NotificationContext);
+  if (!notification) {
+    throw new Error(
+      "useNotification() must be used within a NotificationProvider, which is bundled with EasyUIProvider",
+    );
+  }
+
+  return notification;
+}
+
+export type NotificationPlacementOffset = {
+  /** Top offset */
+  top?: string;
+  /** Right offset */
+  right?: string;
+  /** Bottom offset */
+  bottom?: string;
+  /** Left offset */
+  left?: string;
+};
+
+export type NotificationProviderProps = {
+  /** Child components */
+  children: ReactNode;
+  /** Notification placement offset */
+  notificationPlacementOffset?: NotificationPlacementOffset;
+};
+
+/**
+ * @privateRemarks
+ * The NotificationProvider component bundles everything necessary for an app to
+ * display notifications via the `useNotification` hook. This component shouldn't
+ * be used on its own and is included in the general EasyUI Provider
+ */
+export function NotificationProvider(props: NotificationProviderProps) {
+  const { children, notificationPlacementOffset } = props;
+  const combinedState = useNotificationState();
+
+  const state = {
+    close: combinedState.close,
+    add: combinedState.add,
+    remove: combinedState.remove,
+    pauseAll: combinedState.pauseAll,
+    resumeAll: combinedState.resumeAll,
+    visibleToasts: combinedState.visibleToasts,
+  };
+
+  const notification = {
+    showErrorAlert: combinedState.showErrorAlert,
+    showErrorToast: combinedState.showErrorToast,
+    showPromotionalToast: combinedState.showPromotionalToast,
+    showPromotionalAlert: combinedState.showPromotionalAlert,
+    showNeutralToast: combinedState.showNeutralToast,
+    showNeutralAlert: combinedState.showNeutralAlert,
+    showSuccessToast: combinedState.showSuccessToast,
+    showSuccessAlert: combinedState.showErrorAlert,
```

Comment:
looks like this got mapped wrong:

```suggestion
    showSuccessAlert: combinedState.showSuccessAlert,
```

<img width="1186" alt="image" src="https://github.com/EasyPost/easy-ui/assets/752942/d9935514-6ad8-4cc9-9d73-2ed96819574a">

Reply:
oops, good catch